Formula/calf.rb

```diff
@@ -17,21 +17,6 @@ class Calf < Formula
   depends_on "cairo" => :optional
   depends_on "gtk+" => :optional
 
-
-  head do
-    # Test if linker supports nodelete
-    patch do
-      url "https://github.com/david0/calf/commit/e56702017a3904970f42357a931db949945f43ce.patch"
-      sha256 "89c0765cafe81600774116efd8fa48a6d4878d6913e7f9c05870da7d9f6a3d77"
-    end
-  end
-  stable do
```

Comment:
So I understand the `--HEAD` and default `stable` options better now. 

Although I agree, your stable patch should still be needed (PR merged well after the latest stable release). it's not needed on my machine.  Any idea why?

Steps:

```bash
brew remove calf
brew install calf
# success
```

Reply:
>  doesn't that require an updated `:tag=>` version too to work?

I tried to find version-specific documentation/examples at Homebrew and Formulas but couldn't find one.   I would be happy to add an explicit version check to this with a small amount of direction. 🍻

Reply:
Ok, that should work. We have to keep the stable patches until the  calf releases a new version (creates a new tag). I'm not sure why they are not doing that since years?
Anyway, we can merge it as-it-is any update the stable version later.

About that the change does not make a difference when running brew install: I'm not completely sure, but I guess the "bottle" is still there and rebuild is skipped because of that.
try running with `-s`
```
-s, --build-from-source          Compile formula from source even if a
                                   bottle is provided. Dependencies will still
                                   be installed from bottles if they are
                                   available
```